vMenu/menus/VehicleOptions.cs

```diff
@@ -897,6 +915,17 @@ private void CreateMenu()
                     {
                         Notify.Error(CommonErrors.NoVehicle);
                     }
+                } else if (item == radioStations)
+                {
+                    RadioStation newStation = (RadioStation)Enum.GetValues(typeof(RadioStation)).GetValue(listIndex);
+                    
+                    var veh = GetVehicle();
+                    if (veh != null && veh.Exists())
+                    {
+                        veh.RadioStation = newStation;
+                    }
+
+                    UserDefaults.VehicleDefaultRadio = (int) newStation;
```

Comment:
since you already have the list index value, just use that instead. That way you don't need to convert the enum into an int.